sdk/python/feast/infra/online_stores/online_store.py

```diff
@@ -105,6 +109,180 @@ async def online_read_async(
             f"Online store {self.__class__.__name__} does not support online read async"
         )
 
+    def get_online_features(
```

Comment:
Right now it is, but it's clearly not an ideal one. If we leave everything else aside, just the fact that it exposes read on feature view level rather than a request level means that we need to change it somehow because it's impossible to read data from the online store in a single round-trip. The solution here leaves the abstract methods intact in order to keep things backwards-compatible while also exposing another higher-level method with default implementation that enables other online stores to improve on performance. I don't think keeping it as a strictly abstract interface matters all that much.

Reply:
ok, just curious what was the intention before when the interface was defined? anyway, I think we can go with the new pattern.